Makefile

```diff
@@ -12,7 +12,7 @@ ifeq ($(UNAME_S),Darwin)
 else ifeq ($(UNAME_S),Linux)
     EXTENSION := .so
 	PHP_VERSION := $(shell php -i | grep -Po '(?<=PHP Version => ).*' | uniq)
-    RESTART_COMMAND := sudo systemctl restart php$(PHP_VERSION)-fpm && sudo systemctl restart apache2
+    RESTART_COMMAND := systemctl restart php$(PHP_VERSION)-fpm && systemctl restart apache2
```

Comment:
I don't understand why `sudo` has been removed. These commands require escalated privilege.

Reply:
I was trying to build the php client on a bare ubuntu docker (as root), and this caused a problem as it does not understand sudo. In cases where there is permission required we can just run the make file as sudo -> `sudo make [target]`